.github/pull_request_template.md

```diff
@@ -1,14 +1,34 @@
-< Jira issue link, if applicable >
+_Jira issue link, if applicable_
```

Comment:
Maybe it would be worth it to add here an additional line for the Github issue (e.g. _closes XYZ_). This will ensure that everyone will always remember to mention it and the PR will be bound to the issue?
On the other hand, we don't need that in most cases because we don't have so many issues

Reply:
Will skip this for now